src/guide/migration/emits-option.md

```diff
@@ -0,0 +1,94 @@
+---
+title: emits Option
+badges:
+  - new
+---
+
+# `emits` Option <MigrationBadges :badges="$frontmatter.badges" />
+
+## Overview
+
+Vue 3 now offers an `emits` option similar to the existing `props` option. This option can be used to define the events that a component can emit to its parent.
+
+## 2.x Behavior
+
+In Vue 2, you can define the props that a component received, but you can't declare which events it can emit:
+
+```html
+<template>
+  <p>{{ text }}</p>
+  <button v-on:click="$emit('accepted')">OK</button>
+</template>
+<script>
+  export default {
+    props: ['text']
+  }
+</script>
+```
+
+## 3.x Behavior
+
+Similar to props, the events that the component emit can now be defined with the `emits` option.
+
+```javascript
+<template>
+  <p>{{ text }}</p>
+  <button v-on:click="$emit('accepted')">OK</button>
+</template>
+<script>
+  export default {
+    props: ['text'],
+    emits: ['accepted']
+  }
+</script>
+```
+
+The option also accepts an object notation, which allows the developer to define validators for the arguments that are passed with the emitted event, similar to validators in props definitions.
+
+For more information on this, please read the [API documentation for this feature](../api/options-data.md#emits).
+
+## Migration Strategy
+
+It is highly recommended that you document all of the emitted events of your components this way because of the [removal of the `.native` modifier](./native-modifier-removed.md).
```

Comment:
```suggestion
It is highly recommended that you document all of the emitted events of your components this way because of the [removal of the `.native` modifier](./v-on-native-modifier-removed.md).
```